docs/generated/CommandsInstall.md

```diff
@@ -127,7 +127,7 @@ Alternative to PackageName. This is a list of packages in an xml manifest for Ch
       <package id="alloptions" version="0.1.1"
                source="https://somewhere/api/v2/" installArguments=""
                packageParameters="" forceX86="false" allowMultipleVersions="false"
-               ignoreDependencies="false"
+               ignoreDependencies="false" timeout="1000" force="false"

```

Comment:
@boussaffawalid this file is automatically generated by another part of the Chocolatey, and shouldn't be updated directly.  As you will hopefully notice, this file no longer exists in this repository, as it was recently removed.  The location where you would want to update this would be here:

https://github.com/chocolatey/choco/blob/master/src/chocolatey/infrastructure.app/commands/ChocolateyInstallCommand.cs#L368

Once changed there, the next time we generate the docs, this section would be updated in the generated markdown files.